package.json

```diff
@@ -36,6 +36,7 @@
     "react-redux": "^4.4.1",
     "redux": "^3.3.1",
     "redux-thunk": "^2.0.1",
+    "sparql-client-2": "^0.6.0",
```

Comment:
Yep one of the improvements is that `client-2` allows for Promise chaining, whereas `client` just uses callbacks.
We can either Promisify the callbacks, or use `client-2`